cypress/integration/patch-route.ts

```diff
@@ -10,7 +10,7 @@ const badPatch = {
   id: "i-dont-exist",
 };
 
-const hasText = ($el) => {
+export const hasText = ($el) => {
```

Comment:
I don't see hasText used anywhere else in this diff. Does it need to be exported?